app/src/main/java/ch/sdp/vibester/activity/WelcomeActivity.kt

```diff
@@ -39,6 +39,10 @@ class WelcomeActivity : AppCompatActivity() {
         sendDirectIntent(AuthenticationActivity::class.java)
     }
 
+    fun switchToDownload(view: View) {
+        sendDirectIntent(DownloadActivity::class.java)
+    }
+
```

Comment:
Should be easy to test, currently untested

Reply:
Done!